### ISSUE TITLE
Check & correct dapper ENV variables after #474

### ISSUE THREAD

**Issue by mangelajo** [submariner-io/submariner#477]
See discussion here: 

https://github.com/submariner-io/submariner/pull/474/files#diff-73040b7579d3747d271b2329159c2f83R6

We decided to move on with the patch since it's a considerable cleanup, and then handle that small detail on a separate PR if necessary.
